internal/network/utils.go

```diff
@@ -239,3 +239,31 @@ func GetIPv4Networks(inventory *models.Inventory) (ret []string, err error) {
 func GetIPv6Networks(inventory *models.Inventory) (ret []string, err error) {
 	return getHostNetworks(inventory, func(i *models.Interface) []string { return i.IPV6Addresses })
 }
+
+func areNetworksIdentical(len1, len2 int, f func(int, int) bool) bool {
```

Comment:
Can you give `f` a more meaningful name?
Maybe change this this function to `areNetworkListsIdentical` then `f` could be `areNetworksEqual` or something?

Reply:
Yes